l10n_br_fiscal_dfe/models/dfe.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (C) 2023 KMEE Informatica LTDA
+# License AGPL-3 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+import re
+
+from erpbrasil.assinatura import certificado as cert
+from erpbrasil.transmissao import TransmissaoSOAP
+from nfelib.nfe.ws.edoc_legacy import NFeAdapter as edoc_nfe
+from requests import Session
+
+from odoo import _, api, fields, models
+
+from ..tools import utils
+
+_logger = logging.getLogger(__name__)
+
+
+class DFe(models.Model):
+    _name = "l10n_br_fiscal.dfe"
```

Comment:
aqui o **_name** não teria que ser "l10n_br_fiscal_dfe.dfe "?

Reply:
isso eu acho que infelizmente não tem muita regra nem na OCA nem no Odoo... 
Já tem muita exeçao então não sei se é algo importante tentar respeitar. Por exemplo a gente manteve l10n_br_fiscal.certificate mesmo depois de extrair o módulo.
Mas alguém me corrija se eu tiver errado.

Eu acho que é importante o início do nome do objeto dar uma informação de "scope" mas se não for exactamente o nome do submodulo eu acho que não faz tanta diferença.